main/inc/lib/course.lib.php

```diff
@@ -3906,17 +3907,159 @@ public static function display_courses_in_category($user_category_id, $load_dirs
         return $html;
     }
 
+
+    /**
+     * return html code for diaplaying a course title in the standart view (not the Session view)
+     * @param $inCourseId
+     * @param bool $load_dirs
+     * @return string
+     */
+    public static function display_course_html($inCourseId, $load_dirs = false)
```

Comment:
displayCourseHtml : i've modified an existing function display_course_html
I dont know if it is worth to modify all is occurrences on all the code.